cobigen-templates/templates-oasp4j/src/main/templates/crud_java_server_app/functions.ftl

```diff
@@ -92,11 +92,18 @@
 		}</#if>
 	</#if>
 <#else>
+  /**
+   * @return ${field.name}Id
+   */
   <#if implementsInterface>@Override</#if>
-	public <#if isSearchCriteria>${JavaUtil.boxJavaPrimitives(classObject,field.name)}<#else>${field.type}</#if> <#if field.type=='boolean'>is<#else>get</#if>${field.name?cap_first}() <#if isInterface>;<#else>{
+	public <#if isSearchCriteria>${JavaUtil.boxJavaPrimitives(classObject,field.name)}<#else>${field.type}</#if> <#if field.type=='boolean' || field.type=='Boolean'>is<#else>get</#if>${field.name?cap_first}() <#if isInterface>;<#else>{
```

Comment:
That's not a valid change as it does not comply to the bean standard. If you have an object, you will always ask for getXYZ. This is as you cannot directly interprete it without making sure it is not null. In case of boolean, you can simply ask isXYZ as it will never be null.